bin/local-env/install-wordpress.sh

```diff
@@ -7,6 +7,7 @@ set -e
 WP_DEBUG=${WP_DEBUG-true}
 SCRIPT_DEBUG=${SCRIPT_DEBUG-true}
 WP_VERSION=${WP_VERSION-"latest"}
+HOST=${HOST-"localhost"}
```

Comment:
@swissspidy What you think about this? Make it easy to change the site url. Am I correct in thinking we can use env vars here?

Reply:
Yes, those are env vars.  `HOST` (vs. `WP_HOST`) might be a bit ambiguous, but if it works, it works 👍

Reply:
Fixed.